src/main/java/javapoz24/team3/rental/domain/rental/Rental.java

```diff
@@ -0,0 +1,28 @@
+package javapoz24.team3.rental.domain.rental;
+
+import javapoz24.team3.rental.domain.base.BaseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+import java.net.URI;
+import java.util.Set;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Rental extends BaseEntity {
+
+    private String name;
+    private URI webDomain;
+    private String owner;
+    private URI logoURL;
+    @OneToMany(mappedBy = "rental", fetch = FetchType.EAGER)
```

Comment:
fetch = FetchType.EAGER to zło. 
Przy takic małych aplikacjach może faktycznie nie bedzie widać efektu ale jak pojawi się kilka tysięcy wpisów to zapytania będą chodzić zauważalnie wolniej.
Wiem że kusi ale jednak w środowisku produkcyjnym prawie zawsze jest jednak fetch = FetchType.LAZY, więc nawet dla własnego treningu powinno tak być.

Reply:
Tak, tak wiem, że EAGER to zło, ale jeszcze nigdy nie udało mi się uruchomić LAZY. Zajmę się tym zaraz jak uruchomię pierwszy zestaw encji i powiązań.